content/docs/reference/reference.json

```diff
@@ -554,6 +492,12 @@
     "services": ["proxy"],
     "type": "bool"
   },
+  "databroker-settings": {
+    "id": "databroker",
+    "title": "Databroker Settings",
+    "path": "/databroker",
```

Comment:
Instead of deleting the individual settings / config options, I think it makes sense to keep the entries and link to the consolidated path (e.g. so you can still search for the individual thing).